planning/behavior_path_planner/autoware_behavior_path_avoidance_by_lane_change_module/src/interface.cpp

```diff
@@ -14,8 +14,8 @@
 
 #include "interface.hpp"
 
-#include "autoware_behavior_path_planner_common/interface/scene_module_interface.hpp"
-#include "autoware_behavior_path_planner_common/interface/scene_module_visitor.hpp"
+#include "autoware/behavior_path_planner_common/interface/scene_module_interface.hpp"
+#include "autoware/behavior_path_planner_common/interface/scene_module_visitor.hpp"
```

Comment:
I think there was a mistake in the renaming here 
```suggestion
#include "autoware_behavior_path_planner_common/interface/scene_module_interface.hpp"
#include "autoware_behavior_path_planner_common/interface/scene_module_visitor.hpp"
```

Reply:
Nevermind I did not understand the change to `include/autoware/...`